internal/common/allow.go

```diff
@@ -8,7 +8,7 @@ import (
 	"path/filepath"
 	"regexp"
 
-	"github.com/sirupsen/logrus"
+	"github.com/osbuild/logging/pkg/logrus"
```

Comment:
You will see this in many places, this introduces the mentioned `logrus` proxy so once this PR is merged all logging is done via `log/slog` library. This is temporary.

Reply:
I think this would be good to mention in the commit message.